### ISSUE TITLE
Remove unnecessary alt text for decorative elements on SimpleReport home page

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4032]
## Area or Component
Static site home page - simplereport.gov

## Quick Summary
There are several images on the SimpleReport home page that are purely decorative:
- the blue and brown bubbles behind the animations
- the "mid-wave-top" and "mid-wave-bottom" images used to change the background color in places

## Accessibility Violation
Decorative image has non-empty alt attribute: https://cdc-devtools.dequecloud.com/rule-help/en/alt-text-decorative-inappropriate

## Action Requested
Remove the alt text from the blue bubbles and the wave images

## Screenshots (optional)
Background waves/bubbles seen here have alt text:
![Screen Shot 2022-07-21 at 5 35 13 PM](https://user-images.githubusercontent.com/80282552/180337269-6fc4c1f9-694a-4781-9331-0dd31c336b45.png)
![Screen Shot 2022-07-21 at 5 35 08 PM](https://user-images.githubusercontent.com/80282552/180337282-350f3025-146b-440c-b149-5a844a68d6e5.png)

## Original Deque Issue
https://docs.google.com/spreadsheets/d/16udNpwdJ8sltAULnmSIGgkWVQsfuFvcoymP0-s_QwfE/edit#gid=856291185&range=37:40
